doc/source/guidelines/PyMAPDL_WSL_Guide.rst

```diff
@@ -0,0 +1,423 @@
+  .. _ref_guide_wsl:
+
+
+ANSYS on Windows Subsystem for Linux and Docker
+###############################################
+
+This guide shows you how to use Ansys products, more specifically PyMAPDL, in the Windows Subsystem for Linux (WSL).
+WSL is a compatibility layer for running Linux binary executables natively on Windows 10, Windows 11, and Windows Server 2019 
+(`Wikipedia-WSL <https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux>`_).
+
+This guide walks you through the installation of WSL in Windows, and shows how to use it together with ANSYS, PyMAPDL and docker.
+
+For more information about WSL please visit `What is the Windows Subsystem for Linux? <https://docs.microsoft.com/en-us/windows/wsl/about>`_. 
+
+.. warning:: **Disclaimer:** This guide is still in Alpha state. Please proceed with caution. 
+
+
+.. warning:: This guide hasn't been tested with a VPN connection.
+
+
+Running ANSYS on WSL 
+*********************
+
+Install WSL2
```

Comment:
Why WSL2 here and WSL elsewhere?

Reply:
Very good point..... Thing are that WSL2 is relatively recent so the version 1 is the one implemented by default in Windows. 

Hence, I guess we should probably clarify this a bit more.